marc_to_solr/translation_maps/locations.rb.tmpl

```diff
@@ -80,7 +80,8 @@
  "firestone$prne"=>"Near East Periodicals Collection",
  "firestone$pr"=>"Periodicals Collection",
  "firestone$raas"=>"African American Studies Reading Room: Reserve",
- "firestone$pb"=>"Remote Storage: Supervised use in Firestone A-16-H-3",
+ "firestone$pb"=>
+  "Remote Storage (ReCAP): Supervised use in Firestone A-16-H-3",
```

Comment:
Let me see if I can get some input on what we should say instead of the non-existent A-16-H-3

Reply:
@escowles The change of the non existent 'A-16-H-3' needs to happen in Alma. I dont want to block this pr for this part. The pr is mainly for stokes library.